specification/components/stu3/schemas/responses/cancelReferral/200Response.yaml

```diff
@@ -16,19 +16,19 @@ content:
         value:
           $ref: '../../../examples/cancelReferral/responses/CancelledReferralIntendPrivateWithoutComment.json'
       cancelled-referral-patient-other-reason:
-        summary: Cancelled Referral without Booking with Reason 'Other reason' with Comment (Comment Mandatory)
+        summary: Cancelled Referral without Booking with Reason 'Other' with Comment (Comment Mandatory)
```

Comment:
Are the summary names of the examples included within the stories for documenting endpoints? If so I wonder if these changes also need to be represented in a functional specification?